package.json

```diff
@@ -9,6 +9,7 @@
     "prepare": "npm run clean && tsc",
     "compile": "tsc",
     "lint": "tslint --project tsconfig.json --config tslint.json ",
+    "lint-fix": "tslint --project tsconfig.json --config tslint.json --fix ",
```

Comment:
no real need for this since latest npm and yarn let you add paremeters like

`yarn lint --fix`
`npm run lint -- --fix`

Reply:
Did not know. Cool!

Reply:
It looks like yarn supports this, but NPM does not.